docs/content/docs/connectors/table/kafka.md

```diff
@@ -38,7 +38,7 @@ Dependencies
 {{< sql_download_table "kafka" >}}
 
 The Kafka connector is not currently part of the binary distribution.
-See how to link with it for cluster execution [here]({{< ref "docs/dev/datastream/project-configuration" >}}).
+See how to link with it for cluster execution [here]({{< ref "docs/dev/configuration" >}}).
```

Comment:
This sentence should be in the other sql connectors as well, except filesystem, which is included by default